docs/application/dotnet/guides/user-interface/nui/flex-layout.md

```diff
@@ -34,13 +34,50 @@ Natural size of items are used, which can be different for each item. Additional
 
 The following code shows how to set the `Direction` property using `FlexDirection`:
 
+
+<div id="TabSection1">
```

Comment:
This tab section don't work properly. Please check https://docs1.stg.tizen.org/staging/1521/application/dotnet/guides/user-interface/nui/flex-layout/